src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -587,28 +588,22 @@ public void testCacheAndCheckWhichDoesNotImplementExternalResourceHolderInterfac
         assertWithMessage("ExternalResourceHolder has changed his parent")
                 .that(ExternalResourceHolder.class.isAssignableFrom(HiddenFieldCheck.class))
                 .isFalse();
-        final DefaultConfiguration checkConfig = createModuleConfig(HiddenFieldCheck.class);
-
-        final DefaultConfiguration treeWalkerConfig = createModuleConfig(TreeWalker.class);
-        treeWalkerConfig.addChild(checkConfig);
-
-        final DefaultConfiguration checkerConfig = createRootConfig(treeWalkerConfig);
-        checkerConfig.addProperty("charset", StandardCharsets.UTF_8.name());
-
         final File cacheFile = File.createTempFile("junit", null, temporaryFolder);
-        checkerConfig.addProperty("cacheFile", cacheFile.getPath());
```

Comment:
test method, testCache....... 
We do not have support in BDD inlined config ability to set property of Checker  and Treewalker.
no way to use verifyWithInlineConfigParser, please  use "execute" instead of `verify`

Reply:
sure @romani